### PR TITLE
TEDAPI v3 draft

### PR DESCRIPTION
**Pull request by arnaudjardine**
Work collaboration with OP for new TEDAPI v3

### CODE REVIEW

File: modules/ROOT/attachments/specs/api.yaml [hunk 1]
```diff
@@ -0,0 +1,734 @@
...
+          items:
+            type: string
+            description: The notice languages.
+        noticeAuthor:
+          type: string
+          description: The notice author.
+          example: example@email.eu
+        expectedPublicationDate:
+          type: string
+          description: The notice expected publication date
```
Comment (pcolast) [inline modules/ROOT/attachments/specs/api.yaml]: The notice expected publication date as assigned by the Publications Office based on the OJ S calendar

Comment (arnaudjardine) [reply modules/ROOT/attachments/specs/api.yaml]: Done

**Review by pcolast — CHANGES_REQUESTED** [review]
Dear Arnaud,
It would be helpful to keep the documentation for the existing PROD version (v2) and the next one for v3. For this reason we need to keep the existing files and not to delete them. Api.yaml was modified. Please review and include the modified file in the TEDAPI release. 
Thank you.
Stefan

**Comment by bertrand-lorentz** [comment]
@pcolast : I guess this pull request will not be merged ? If that is the case, then please close it.

**Comment by pcolast** [comment]
Closing this pull request as it will not be merged. 
api.yaml taken from the TEDAPI 2.0.0 release contains the latest changes.

**Pull request closed without merging**